.github/workflows/ci.yml

```diff
@@ -9,18 +9,16 @@ on:
 jobs:
   xcode-build:
     name: Xcode Build
-    runs-on: macOS-11
+    runs-on: macOS-14
     strategy:
       matrix:
-        platform: ['iOS_14', 'iOS_13']
```

Comment:
In practice I've never seen a difference across iOS versions with Paralayout's test suite, so not too concerned about having more limited iOS version coverage. I think the advantage of having 2x and 3x scales here outweighs the version matrix.